rp_asos/views.py

```diff
@@ -15,3 +32,13 @@ def post(self, request, project_id, tz_code, *args, **kwargs):
             chain.delay(project_id, tz_code)
 
         return HttpResponse(status=return_status)
+
+
+class StartScreeningRecordCheckView(APIView):
+    def post(self, request, org_id, *args, **kwargs):
```

Comment:
What is going to trigger this view? Redcap? Do we want to include `get` if we want to trigger it from RP?

Reply:
I will trigger it from Rapidpro. You can select GET/POST when setting up a webhook in RP.